Objective-C/CBLReplicator.mm

```diff
@@ -73,6 +73,7 @@ @interface CBLReplicator ()
 @implementation CBLReplicator
 {
     dispatch_queue_t _dispatchQueue;
+    dispatch_queue_t _otherDispatchQueue; // TODO: Naming???
```

Comment:
How about _conflictResolverDispatchQueue?

Reply:
this can be used for any thing other than normal database serial queue right. so if we name only for conflict resolution, that might only be used for one purpose. Your thoughts?

NB: also how about `_concurrentQueue?`

Reply:
i named it `_concurrentQueue ` for now, do you have objections?

Reply:
updated name to `_conflictResolverQueue `, till we have other purpose for concurrent operations.